internal/driver/driver.go

```diff
@@ -14,65 +14,58 @@ import (
 	"sync"
 
 	"github.com/edgexfoundry/device-opcua-go/internal/server"
-	sdkModel "github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
-	"github.com/edgexfoundry/device-sdk-go/v2/pkg/service"
-	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/logger"
-	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+	"github.com/edgexfoundry/device-sdk-go/v3/pkg/interfaces"
+	sdkModel "github.com/edgexfoundry/device-sdk-go/v3/pkg/models"
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/models"
 )
 
 var once sync.Once
 var driver *Driver
 
 // Driver struct
 type Driver struct {
-	Logger    logger.LoggingClient
-	AsyncCh   chan<- *sdkModel.AsyncValues
 	mu        sync.Mutex
 	serverMap map[string]*server.Server
+	sdk       interfaces.DeviceServiceSDK
 }
 
 // NewProtocolDriver returns a new protocol driver object
-func NewProtocolDriver() sdkModel.ProtocolDriver {
+func NewProtocolDriver() interfaces.ProtocolDriver {
 	once.Do(func() {
 		driver = new(Driver)
 	})
 	return driver
 }
 
 // Initialize performs protocol-specific initialization for the device service
-func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *sdkModel.AsyncValues, deviceCh chan<- []sdkModel.DiscoveredDevice) error {
-	d.Logger = lc
-	d.AsyncCh = asyncCh
+func (d *Driver) Initialize(sdk interfaces.DeviceServiceSDK) error {
+	d.sdk = sdk
+
+	// Define custom API endpoints
+	if err := d.sdk.AddRoute("/api/v3/call", handleMethodCall, http.MethodPost); err != nil {
```

Comment:
What is this `/call` for?

Reply:
Calling OPC UA methods associated with a given device. It was in the previous version - I just updated `/v2/` to `/v3/`. See [Using Methods](https://github.com/SE-I-T-Digital/device-opcua-go/tree/minnesota#using-methods) in the README